wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -312,16 +312,36 @@ namespace :wagn do
     desc "copy files from template database to standard mod and update cards"
     task :copy_mod_files => :environment do
 
-      mod_name = '05_standard'
-      template_files_dir = "#{Wagn.root}/files"
-      standard_files_dir = "#{Cardio.gem_root}/mod/#{mod_name}/file"
-
-      FileUtils.remove_dir standard_files_dir, force=true
-      FileUtils.cp_r template_files_dir, standard_files_dir
-
+      source_files_dir = "#{Wagn.root}/files"
       # add a fourth line to the raw content of each image (or file) to identify it as a mod file
       Card::Auth.as_bot do
         Card.search( :type=>['in', 'Image', 'File'], :ne=>'' ).each do |card|
+          
+          if card.attach_mod
+            puts "skipping #{card.name}: already in code"
+            next
+          else
+            puts "working on #{card.name}"
+          end
+
+          base_card = card.cardname.junction? ? card.left : card
+          raise "need codename for file" unless base_card.codename.present?
+          
+          mod_name = base_card.type_id==Card::SkinID ? '06_bootstrap' : '05_standard'
```

Comment:
this code won't win any beauty contests, but it does handle converting file representation from id-based to codename-based pretty well.